FlowCrypt/Core/CoreTypes.swift

```diff
@@ -76,6 +102,12 @@ struct UserId: Encodable {
     let name: String;
 }
 
+extension UserId {
+    func toMime() -> String {
+        return "\(self.name) <\(self.email)>"
```

Comment:
There should be proper escaping in place (currently there is not), maybe the imap library has a method for this.